_pages/en/esdc-it-strategy.md

```diff
@@ -16,32 +16,25 @@ The [mission of Employment and Social Development Canada (ESDC)](https://www.can
+Internally within ESDC, the Innovation, Information and Technology Branch (IITB) established the [IITB Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx), containing 26 service improvement and modernization activities to improve its Information Management/Information Technology (IM/IT) service offerings.
 
 In addition, ESDC has started investing in a [Business Delivery Modernization](https://www.canada.ca/en/employment-social-development/corporate/reports/departmental-plan/2019-2020/planned-results.html#h2.04) effort that targets a complete business process and technology renewal for Employment Insurance, Old Age Security and the Canada Pension Plan.
 
 ## Problem Statement
 
-IT and Business Continuity are key organizational risks for a large organization like ESDC.
-Both risks have impacts to Information/Knowledge Management (Data), Privacy/Safeguarding Personal Information, Fraud and Service Delivery.
-
 As explained in [ESDC's Medium-Term Planning](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Briefings/MTP-Initiative-Mapping_2018.docx), ESDC operates a portfolio of approximately 500 applications, 101 of which are mission critical.
 These IT systems can no longer meet the needs of ESDC programs due to system aging and too much unaddressed technical debt.
 Some new government regulations cannot be implemented because ESDC IT systems are unable to respond to those changes.
 
-Government of Canada historical records are evidence that the conventional method of doing IT promotes risks instead of reducing them.
-For example, the [July 2018 report of the Standing Senate Committee on National Finance](https://sencanada.ca/content/sen/committee/421/NFFN/reports/NFFN_Phoenix_Report_32_WEB_e.pdf) about the Phoenix Pay System project states: "Instead of realizing $70 million in annual savings by centralizing pay operations, the government will incur approximately $2.2 billion in unplanned expenditures".
+In addition, with the emergence of digital government that is reviewing business processes, policy, and delivery, the demand on IT is expected to increase exponentially. This [exposes the ubiquitous nature of technology in government service delivery](https://www.oag-bvg.gc.ca/internet/English/parl_oag_200611_03_e_14971.html) and that IT can no longer be seen as a back-office function, it has real repercussions to Canadians. IT can no longer be considered a back office function or a "company within a company" as it has real repercussions to Canadians.
```

Comment:
> demand on IT is expected to increase exponentially

Is it not already increasing? Maybe we could reword to highlight that it's already happening.

Reply:
@gabrielcossette reworded to say
"the demand on IT is expected continue increasing exponentially"